docs/packages/xstate-vue/index.md

```diff
@@ -64,6 +67,94 @@ A [Vue comosition function](https://vue-composition-api-rfc.netlify.com/) that i
 - `send` - A function that sends events to the running service.
 - `service` - The created service.
 
-### `useService(service)` (TODO)
+### `useService(service)`
+
+A [Vue comosition function](https://vue-composition-api-rfc.netlify.com/) that subscribes to state changes from an existing [service](TODO).
+
+**Arguments**
+
+- `service` - An [XState service](https://xstate.js.org/docs/guides/communication.html).
+
+**Returns** `{current, send}`:
+
+- `current` - Represents the current state of the service as an XState `State` object.
+- `send` - A function that sends events to the running service.
+
+### `useActor(actor)`
+
+A [Vue comosition function](https://vue-composition-api-rfc.netlify.com/) that subscribes to messages (events) from actors, and can send messages (events) to actors.
+
+**Arguments**
+
+- `actor` - An actor-like object, which has `.subscribe(listener)` and `.send(event)` methods.
+
+**Returns** `{current, send}`:
+
+- `current` - Represents the current message sent from the actor.
+- `send` - A function that sends events to the actor.
```

Comment:
For now, I'm not going to include the `useActor` function in the PR because I can't get it to work properly with types, so we can leave that out for now.

Reply:
yeah you can remove it.